pkg/cache/cache.go

```diff
@@ -67,6 +67,8 @@ type RemoteCache struct {
 	records map[digest.Digest]*Item
 	// size is the cache record capacity of target layers.
 	size int
+	// updateCount records the number of new caches added in one conversion.
+	updateCount int
```

Comment:
Is it necessary?

Reply:
I think it is necessary to calculate the cached layer size.